frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeSection.tsx

```diff
@@ -108,6 +121,7 @@ const ServingRuntimeSizeSection: React.FC<ServingRuntimeSizeSectionProps> = ({
             setAcceleratorState={setAcceleratorState}
             supportedAccelerators={supportedAccelerators}
             resourceDisplayName="serving runtime"
+            infoContent="Be sure appropriate tolerations are in place before adding Accelerator to your model server"
```

Comment:
Ok, this phrase sounds about right, but @kaedward @vconzola can you help us out here please?

Reply:
@vconzola now that you are around, can we check this? we adapted the label to accelerators. cc @kaedward

Reply:
@lucferbux @vconzola 
"Ensure that appropriate tolerations are in place before adding an accelerator to your model server." 

^^This would be the correct phrasing from a content perspective... But is the field label still "GPUs"? What are the appropriate tolerations? This part has confused me since it was added, but now that it's changing again I'm totally lost haha. If this suggestion sounds right to you, feel free to use it, but I cannot verify the *technical* accuracy of it :P